packages/tests/package.json

```diff
@@ -14,16 +21,17 @@
         "test": "NODE_OPTIONS=\"--openssl-legacy-provider ${NODE_OPTIONS:-}\" jest --verbose"
     },
     "dependencies": {
-        "@datadog/build-plugins-core": "workspace:*",
-        "@datadog/esbuild-plugin": "workspace:*",
-        "@datadog/webpack-plugin": "workspace:*",
+        "@datadog/esbuild-plugin": "workspace:0.0.13-4",
+        "@datadog/webpack-plugin": "workspace:0.0.13-4",
```

Comment:
To note: you shouldn't used pinned version, but `workspace:*`

yarn will use the version automatically of the other package on publish 
And if you publish them in order, it should be okay (this is what I do with my packages and it works amazingly good)